<!DOCTYPE html>
<html lang="en" dir="ltr" class="client-nojs">

<head>
    <meta charset="UTF-8" />
    <title>Protocol</title>
    <link rel="stylesheet" type="text/css" href="css/nav.css">
    <link rel="stylesheet" type="text/css" href="css/painter.css">
    <link rel="stylesheet" type="text/css" href="./css/new1.css">
    <link rel="stylesheet" type="text/css" href="./css/page/protocol.css">
    <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.5.1.min.js"></script>
    <script>
        window.addEventListener('scroll', function() {
            let top = window.scrollY
            console.log(top)
            if (top > 200) {
                $('.main').css('transform', 'translateY(-300px)')
            } else {
                $('.main').css('transform', 'translateY(0%)')
            }
        })
    </script>
</head>

<body>
    <div id="loading">
        <div class='loadingX' style="background-color: #abc2ea;">
            <img src="./image/lodding_3.gif" height=100% width=100%>
        </div>
    </div>

    <script>
        document.onreadystatechange = function() {

            if (document.readyState === "complete") {

                setTimeout(
                    function() {
                        // document.querySelector("#loading").style.opacity = "0%";
                        document.querySelector("#loading").style.display = "none";
                        document.querySelector("#globalWrapper").style.display = "block";
                    }, 3000)
            }
        }
    </script>

    <div id='globalWrapper'>
        <div id="content" class="mw-body" role="main" style="background-color:  white">


            <div id="HQ_page">
                <div id="bodyContent">

                    <div id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr">

                        <section class="menu-section-area">

                            <!-- Navigation -->
                            <nav class="menu">
                                <div style="height: 0px;width: 0px;position: relative;">
                                    <img src="./image/logo.png" width="50px" height="50px" style="position:absolute;top:-14px;">
                                </div>

                                <ol>
                                    <li class="menu-item"><a href="./index.html">Home</a></li>

                                    <li class="menu-item">
                                        <a href="#0">Project
                                            <svg version="1.1" class="plus-icon" xmlns="http://www.w3.org/2000/svg"
                                                xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 18 18">
                                                <line fill="none" stroke-width="4" stroke-linecap="round"
                                                    stroke-miterlimit="10" x1="10" y1="9" x2="17" y2="9" />
                                                <line fill="none" stroke-width="4" stroke-linecap="round"
                                                    stroke-miterlimit="10" x1="9" y1="9" x2="9" y2="1" />
                                                <g id="lineGroup_1">
                                                    <line fill="none" stroke-width="4" stroke-linecap="round"
                                                        stroke-miterlimit="10" x1="1" y1="9" x2="8" y2="9" />
                                                    <line fill="none" stroke-width="4" stroke-linecap="round"
                                                        stroke-miterlimit="10" x1="9" y1="17" x2="9" y2="9" />
                                                </g>
                                            </svg>
                                        </a>
                                        <ol class="sub-menu">
                                            <li class="sub-menu-item"><a href="./description.html">Description</a></li>
                                            <li class="sub-menu-item"><a href="./design.html">Design</a></li>
                                            <li class="sub-menu-item"><a href="./results.html">Results</a></li>
                                            <li class="sub-menu-item"><a href="./supplement.html">Supplement</a></li>

                                            <li class="sub-menu-item"><a href="./protocol.html">Protocol</a></li>
                                        </ol>
                                    </li>
                                    <li class="menu-item"><a href="./lab_notebook.html">Lab notebook</a></li>
                                    <li class="menu-item">
                                        <a href="#0">Team
                                            <svg version="1.1" class="plus-icon" xmlns="http://www.w3.org/2000/svg"
                                                xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 18 18">
                                                <line fill="none" stroke-width="4" stroke-linecap="round"
                                                    stroke-miterlimit="10" x1="10" y1="9" x2="17" y2="9" />
                                                <line fill="none" stroke-width="4" stroke-linecap="round"
                                                    stroke-miterlimit="10" x1="9" y1="9" x2="9" y2="1" />
                                                <g id="lineGroup_1">
                                                    <line fill="none" stroke-width="4" stroke-linecap="round"
                                                        stroke-miterlimit="10" x1="1" y1="9" x2="8" y2="9" />
                                                    <line fill="none" stroke-width="4" stroke-linecap="round"
                                                        stroke-miterlimit="10" x1="9" y1="17" x2="9" y2="9" />
                                                </g>
                                            </svg>
                                        </a>
                                        <ol class="sub-menu">
                                            <li class="sub-menu-item"><a href="./team.html">Team</a></li>
                                            <li class="sub-menu-item"><a href="./collaboration.html">Collaboration</a>
                                            </li>

                                        </ol>
                                    </li>



                            </nav>
                            <!-- Navigation -->

                        </section>
                        <div class="img1"></div>

                        <div class="lnav">
                            <ul class="navul" style="margin-left: 0;padding: 0;">
                                <li id="nav1" class="uli cntli">
                                    <a href="#h1">Materials</a>
                                </li>
                                <li id="nav2" class="uli cntli">
                                    <a href="#h2">Methods</a>
                                </li>
                                <li id="nav3" class="uli cntli" style="height: 81px;">
                                    <a href="#h3">Methods of PACE</a>
                                </li>
                            </ul>
                        </div>



                        <div class="content">
                            <div class="main" style="color: black;word-wrap: break-word;word-break: normal;">
                                <aside>
                                    <div class="p1" style="word-wrap:break-word;word-break: normal;">

                                        </br>


                                    </div>

                                    <div class="p2" id="h1">
                                        </br>
                                        <h1 style="text-align:left;line-height: 30px; ">
                                            Materials</h1>
                                        <h2 style="font-size: 20px; margin-top: 30px;color: black;">
                                            Biological materials
                                        </h2>
                                        <p style="text-indent: 0em;">
                                            <i>E. coli</i> S2060 (Addgene, cat. no. 105064)
                                            <br /><i>E. coli</i> S2208, pJC175e was transformed into S2060 to obtain the strain.

                                            <br /><i>E. coli</i> C321 ΔA muts-T7

                                        </p>

                                        <h2 style="font-size: 20px; margin-top: 30px;color: black;text-align:left;">
                                            Plasmids
                                        </h2>
                                        <p style="text-indent: 0em;">
                                            sfGFP Y66-S205C
                                            <br />p15A-PBAD-Gen2-tRNACUA opt
                                            <br />Mutagenesis plasmid MP6 (Addgene, cat. no. 69669)
                                            <br />pJC175e (Addgene, cat. no. 79219)
                                            <br />Accessory plasmid, including a tRNA sequence that recognises the TAG codon and the
                                            <I>gⅢ</I> and LuxAB genes controlled by the T7 promoter.
                                            <br />Complementary plasmid, including the T7 RNAP which contains two amber termination mutations.
                                            <br />Selection plasmid, derived from bacteriophage M13 (V00604.2, ATCC 15669-B1) with the following modifications:
                                            <br />Removal of <i>gⅢ</i>
                                            <br /> Substitution of <i>gⅢ</i> RBS with synthetic RBS
                                            <br /> Insertion of gene of interest immediately after synthetic RBS:AAAGAGGAGAAA

                                            <br /> Insertion of the following sequence, containing an artificial promoter and RBS for production of downstream
                                            <I>gVI</I>：
                                            <br />GGCTCTAGAAGGAGATTTTCAACATGCTCCCTCAATCGGTTGAATGTCGCCCTTTTGTCTTTGGCGCTGGTAAACCATATGAATTTTCTATTGATTGTGACAAAATAAACTTATTCCGTGGTGTCTTTGCGTTTCTTTTATATGTTGCCACCTTTATGTATGTATCTTCTACGTTTGCTAACATACTGCGTAATAAGGAGTCTTAATC

                                        </p>
                                        <div class="notebook_cell" style="text-align:center;">

                                            <h2 style="font-size: 20px; margin-top: 30px;color: black;text-align:left;">
                                                Primers</h2>
                                            </br>

                                            <table style="border:black 2px solid;text-align: center;align-items: center;margin: 0 auto;" border="1">
                                                <tr>
                                                    <th>Name</th>
                                                    <th>Sequence 5'-3'</th>
                                                </tr>

                                                <tr>
                                                    <td>Gen 2-F</td>
                                                    <td>GCTAACAGGAGGAATTAACCATGGACGAATTTGAAATGATAAAGAG</td>
                                                </tr>
                                                <tr>
                                                    <td>Gen 2-R</td>
                                                    <td>CGGCATAGCGAAACTTAAAATTATAATCTCTTTCTAATTGGCTCTAAAATC</td>
                                                </tr>
                                                <tr>
                                                    <td>Vector-F</td>
                                                    <td>TTTTAAGTTTCGCTATGCCGG</td>
                                                </tr>
                                                <tr>
                                                    <td>Vector-R</td>
                                                    <td>GGTTAATTCCTCCTGTTAGCCCA</td>
                                                </tr>
                                                <tr>
                                                    <td>Reverse PCR-RBS-F</td>
                                                    <td>CACCTGCAGGTCTTTCTCCCTATAGTGAGTCGTATTAGTCGGCC</td>
                                                </tr>
                                                <tr>
                                                    <td>Reverse PCR-PROK-R</td>
                                                    <td>AGCAGGCTTTTTTGCATGTCATGCCAGTTCTAGCATAACCCC</td>
                                                </tr>
                                                <tr>
                                                    <td>G3-RBS-F</td>
                                                    <td>GAGAAAGACCTGCAGGTGCAGTAAGGAG</td>
                                                </tr>
                                                <tr>
                                                    <td>G3-PROK-R</td>
                                                    <td>AGGCATTTGAGAAGCACATTAGGTATATTCCGTGTGGTACTT</td>
                                                </tr>
                                                <tr>
                                                    <td>PROK-F</td>
                                                    <td>TGTGCTTCTCAAATGCCTGAG</td>
                                                </tr>
                                                <tr>
                                                    <td>PROK-R</td>
                                                    <td>CATGCAAAAAAGCCTGCTCGTTG</td>
                                                </tr>
                                                <tr>
                                                    <td>T7-P-F</td>
                                                    <td>CTACATAAATCAAACATAAGGAGGTAACCATGAACACGATTAACATCGCTAAGAAC</td>
                                                </tr>
                                                <tr>
                                                    <td>T7-P-R</td>
                                                    <td>GCTCTTCAGCCTTACGCGAACGCGAAGTCCG</td>
                                                </tr>
                                                <tr>
                                                    <td>AP-F</td>
                                                    <td>TCGCGTAAGGCTGAAGAGCTCATGCCAGTTCTAGCATAACCC</td>
                                                </tr>
                                                <tr>
                                                    <td>AP-R</td>
                                                    <td>TTATGTTTGATTTATGTAGTAGACTAGAAGAGCGTCGGCCTTACTTGCTAGCAG</td>
                                                </tr>
                                                <tr>
                                                    <td>AP-T7-F</td>
                                                    <td>AAGCAGAAGGCCATCCTGAC</td>
                                                </tr>
                                                <tr>
                                                    <td>AP-T7-R</td>
                                                    <td>GCGACACGGAAATGTTGAATAC</td>
                                                </tr>
                                                <tr>
                                                    <td>GOI-F</td>
                                                    <td>TAATGGAAACTTCCTCATGAAAAAGTCTTTAG</td>
                                                </tr>
                                                <tr>
                                                    <td>GOI-R</td>
                                                    <td>ACAGAGAGAATAACATAAAAACAGGGAAGC</td>
                                                </tr>
                                                <tr>
                                                    <td>pJC175e-Check-F</td>
                                                    <td>GCGGTATCATCAACAGGCTTA </td>
                                                    </td>
                                                </tr>
                                                <tr>
                                                    <td>pJC175e-Check-R</td>
                                                    <td>CGTTCCAGTAAGCGTCATACAT</td>
                                                </tr>
                                                <tr>
                                                    <td>CP-12S-F</td>
                                                    <td>CGACTTCTAGGACATCGAACTGGCTGCTATCC</td>
                                                </tr>
                                                <tr>
                                                    <td>CP-12S-R</td>
                                                    <td>CGATGTCCTAGAAGTCGTTCTTAGCGATGTTAATCG</td>
                                                </tr>
                                                <tr>
                                                    <td>CP-203-F</td>
                                                    <td>GTGGTCTTAGTGGCATAAGGAAGACTCTATTCATG</td>
                                                </tr>
                                                <tr>
                                                    <td>CP-203-R</td>
                                                    <td>TATGCCACTAAGACCACGCCTCGCCACC</td>
                                                </tr>



                                            </table>
                                        </div>
                                        </br>
                                        <h2 style="font-size: 20px; margin-top: 30px;color: black;">
                                            Reagents </h2>

                                        <p style="text-indent: 0em;">
                                            2× YT Medium (Solarbio, cat. no. LA2470)
                                            </br>YEAST EXTRACT (OXOID, cat. no. LP0021) TRYPTONE (OXOID,cat. no. LP0042) NaCl (HUSHI,cat. no. 10019318)
                                            </br> SOC media (Solarbio, cat. no. LA2500)
                                            </br>X-Gal (Solarbio, cat. no. R0404)
                                            </br>Agar (Solarbio, cat. no. A8190)
                                            </br>Ampicillin (Solarbio, cat. no. A7094)
                                            </br>Chloramphenicol (Solarbio, cat. no. C8050)
                                            </br> Tetracycline (Solarbio, cat. no. IT0130)
                                            </br>Streptomycin (Solarbio, cat. no. S8290)
                                            </br>L-arabinose (Solarbio, cat. no. A8060)
                                            </br>Kanamycin (Solarbio, cat. no. K8020)
                                            </br>lucose (Solarbio, cat. no. G8150)
                                            </br> Anhydrotetracycline (Solarbio, cat. no. IA5330)
                                            </br>IPTG (Isopropyl β-D- Thiogalactopyranoside ) (Solarbio, cat. no. I8070)

                                            </br>Ascorbic acid (Solarbio, cat. no. A8100
                                            </br>LBL medium
                                            </br>TRYPTONE 10 g/L, YEAST EXTRACT 5 g/L, NaCl 5 g/L, 0.5 mg/ml ascorbic acid, pH=7.45, 33 ug/ml Chloramphenicol.
                                            </br>DAM medium
                                            </br>5g/L TRYPTONE, 2.5g/L yeast extract, 5g/L NaCl, 0.5mg/ml ascorbic acid, 10mM/L L-DOPA, 25uM/L Kanamycin, 2.96g/L IPTG.
                                            </br>SapⅠ (New England BioLabs, cat. no. R0569S)
                                            </br>T4 DNA ligase (New England BioLabs, cat. no. M0202V).
                                            </br>FastPure Gel DNA (Nanjing Vazyme, cat. DC301-01)
                                            </br>FastPure Plasmid Mini kit (Nanjing Vazyme, cat. DC201-01)
                                            </br>DH5α Competent cell (Nanjing Vazyme, cat. C502-02)
                                            </br>ClonExpress MultiS One Step Cloning Kit (Nanjing Vazyme, cat. C113-01)

                                            </br>ClonExpress II One Step Cloning Kit (Nanjing Vazyme, cat. C112-01)

                                            </br> 2×YT liquid media
                                            </br> Add 2×YT media powder to a final concentration of 31 g/L in water. Mix to dissolve and then autoclave to sterilize at 121.0 °C. Store at room temperature indefinitely.
                                            </br> 2×YT agar
                                            </br>Add 2×YT media powder and agar to a final concentration of 31 g/L and 1.5% (wtol), respectively, in water. Autoclave to sterilize at 121.0 °C. Store at room temperature in the dark for up to 6 months.
                                            </br>2×YT top agar
                                            </br>Melt 2×YT agar using a microwave until completely liquid. Dilute hot 2×YT agar with room temperature 2×YT liquid media to a final agar concentration of 0.5% or 0.6%(wtol) and mix thoroughly.
                                            </br>LB liquid media
                                            </br>Add YEAST EXTRACT, TRYPTONE, NaCl to a final concentration of 5 g/L,10 g/L,10 g/L in water, respectively.
                                            </br>LB agar
                                            </br>Add YEAST EXTRACT, TRYPTONE, NaCl, agar to a final concentration of 5 g/L, 10g/L, 10g/L, 1.5% (wtol), respectively, in water.
                                            </br> 2×TSS
                                            </br>Add MgCl2, PEG 3350 and DMSO to LB media at final concentrations of 20 mM, 10% wtol and 5% volol, respectively, and sterilize using a Corning bottle-top vacuum filter system. Store at 4 °C indefinitely.

                                            </br>5× KCM solution
                                            </br>Add KCl, CaCl2 and MgCl2 to water at final concentrations of 100 mM, 30 mM and 50 mM, respectively. Store at room temperature indefinitely.

                                            </br> DRM
                                            </br>To 900 ml of water, add Harvard Custom Media A (18.1 g) and TWEEN-20 or TWEEN-80 (1 mL) and then autoclave to sterilize at 121.0℃. Allow to cool completely. Separately, add Harvard Custom Media C (5.9 g),
                                            5 µl of 0.1 M CaCl2 solution and 6 µl of Trace Metals solution to 100 mL of water and sterilize using a Corning bottle-top vacuum filter system. Add the Harvard Custom Media C solution to the cooled Harvard
                                            Custom Media A solution to make complete DRM. Store in the dark at 4 °C for up to 1 year.
                                        </p>
                                        <h2 style="font-size: 20px; margin-top: 30px;color: black;">
                                            Equipment
                                        </h2>
                                        <p style="text-indent: 0em;">
                                            Biological shaker (Shanghai Zhichu Instruments, cat. no. ZHTY-50ES)
                                            <br />Benchtop centrifuge (Thermo Fisher Scientific, cat. no. 75007201)

                                            <br />Nano-100 microspectrophotometer(Hangzhou Allsheng Instruments, cat. no. AS-11010-00)

                                            <br />UV-Vis spectrophotometer(Thermo Fisher Scientific, cat. no. Themo Genesys10s)

                                            <br /> Electrophoresis(Beijing Liuyi, cat. no. DYY-6C)


                                            <br />Absorbance Microplate Reader(TENCAN, cat. no.Tecan Infinite M200 NanoQuant)

                                            <br />Isolated thermostatic incubator (Shanghai SENXIN , cat. no. GNP-9080)
                                            <br />Gene Explorer(Bioer technology, cat. TC-96/G/H(b)B)
                                            <br />13 mm, 0.22-μm PVDF syringe filter(0.22 µm; Merck Millipore Ltd., cat. no. PR05538)

                                            <br />Gel imaging system(CLiN Qingxiang, cat. no. GenoSens2100)

                                        </p>
                                    </div>



                                    <div class="p3 h2" id="h2">
                                        </br>
                                        <h1 style="text-align:left;line-height: 30px; ">
                                            Methods</h1>
                                        <h2 style="font-size: 20px; margin-top: 30px;color: black;">
                                            Preparation
                                        </h2>
                                        <h3 style="font-size: 17px; margin-top: 30px;color: black;">Preparation of competent cells</h3>

                                        <p style="text-indent:0em;">
                                            Frozen strains at -70℃ were inoculated into LB liquid medium（For S2060 and S2208 strains, use 2×YT liquid medium） by streaking method, labeled, and cultured overnight at 37℃. On the second day, a single colony was picked from the plate and inoculated
                                            into a test tube containing 5ml LB liquid medium（For S2060 and S2208 strains, use 2×YT liquid medium） and incubated at 37℃ overnight by shaking; on the next day, 1mL of bacterial solution was inoculated into
                                            a 500mL flask containing 100mL LB liquid medium（For S2060 and S2208 strains, use 2×YT liquid medium ） and incubated at 37℃ for about 4-6 hours (200-300 /min) by shaking violently. When the colony OD600 reaches
                                            0.4-0.6, the bacterial solution is poured into a 50mL centrifuge tube under aseptic condition and centrifuged at 4000g for 10 minutes at 4℃. Discard the supernatant and add 2mL of pre-cooled LB liquid medium
                                            to the centrifuge tube, mix gently, and suspend the bacteria. Add 2mL of pre-cooled 2xTSS to the centrifuge tube, mix thoroughly, divide it into 100μL parts, put them into 1.5mL centrifuge tube, and freeze them
                                            with dry ice or liquid nitrogen. Store at -80 ℃.
                                        </p>

                                        </br>
                                        <h3 style="font-size: 17px; margin-top: 30px;color: black;">Mention of plasmid
                                        </h3>

                                        <p style="text-indent:0em;">
                                            According to the kit instructions for Nanjing Vazyme. </p>

                                        </br>
                                        <h3 style="font-size: 17px; margin-top: 30px;color: black;">PCR fragment and vector
                                        </h3>

                                        <p style="text-indent:0em;">
                                            The obtained plasmids are subjected to PCR using different programs and primers.
                                            <br /> PCR Gen 2 fragment with Gen 2-F, Gen 2-R<br /> PCR pBAD33 vector with Vector-F, Vector-R<br /> After agarose gel electrophoresis, we extract DNA from the gel. The products are stored at -20 ℃ for subsequent
                                            use.
                                        </p>
                                        <h2 style="font-size: 20px; margin-top: 30px;color: black;">
                                            Setting mutation conditions
                                        </h2>
                                        <h3 style="font-size: 17px; margin-top: 30px;color: black;">Error-prone PCR</h3>

                                        <p style="text-indent:2em;">
                                            Use StarMut Random Mutagenesis Kit. Multiple small system (20μL) amplification pre-experiments were carried out, setting a series of annealing temperature gradients and StarMut Enhancer gradients. According to the PCR results and the mutation rate in
                                            the instructions. Final temperature (51.3℃) and Enhancer volume (6μL) were determined. </p>

                                        <h3>Site-directed mutagenesis</h3>

                                        <p style="text-indent:2em;">
                                            With the help of two bioinformatic tools: fpocket and CAVER, we spot four mutational residues: Y32L, A67S, H70N, A167Q.Three pairs of degenerate primer (IUBcode)were designed to randomly introduce mutations in Gen 2, in which A67&H70 were too close to
                                            be combined to one fragment.
                                        </p>

                                        <h2 style="font-size: 20px; margin-top: 30px;color: black;">
                                            Mutation
                                        </h2>
                                        <h3 style="font-size: 17px; margin-top: 30px;color: black;">Error-prone PCR</h3>

                                        <p style="text-indent:2em;">
                                            Using the error-prone PCR kit purchased from Gene-star company with (template plasmid 1, 2×StarMut Random PCR Mix 10μL, Gen 2-F, Gen 2-R 1μL each, Star Mut enhancer 6μL, sterile water 1μL, 53.1℃) conditions for error-prone PCR on MjTYR-Gen2 fragments.
                                            The PCR products were verified by electrophoresis and purified using a kit purchased from Vazyme. The linearized vector and error-prone PCR fragments were then subjected to homologous recombination using homologous
                                            recombination reagents purchased from Vazyme. </p>


                                        <h3 style="font-size: 17px; margin-top: 30px;color: black;">Site-directed mutagenesis
                                        </h3>

                                        <p style="text-indent:2em;">
                                            The whole pBAD33-Gen 2-Angew tRNA plasmid was divided into three parts and then amplified by three pairs of degenerate primers and 2 × Phanta Max Master Mix (Dye Plus) (Vazyme Biotech Co.,Ltd). The consequent three fragments with randomly introduced mutations
                                            were then combined with ClonExpress MultiS One Step Cloning Kit (Vazyme), which is a cloning kit based on homologous recombination. E.coli C321 with pET28a-GfpS205C were used for subsequent chemical transformation.
                                        </p>

                                        <h2 style="font-size: 20px; margin-top: 30px;color: black;">
                                            Transformation
                                        </h2>


                                        <p style="text-indent:2em;">
                                            Competent cells were thawed on ice. Pre-cooled plasmid mixture was added (each plasmid should
                                            < 100 ng; Each transformation of up to 3 plasmids), fully mixed after 30min on ice, 42℃ water bath, heat shock 90s(For S2060 and S2208 strains, heat shock 75s). Put back on ice and stand for 2-3min; Add 500 microliters of LB liquid medium(For S2060 and
                                                S2208 strains, use SOC medium), shake at 37℃, 200-300rpm/min, 1h; The bacterial solution was centrifuged at 8000rpm for 1min, the appropriate supernatant was removed, and then resuspended and coated in LB solid
                                                medium(For S2060 and S2208 strains, use 2×YT solid medium) containing appropriate antibiotics and 1.5% agar at 37℃ for 16-18h. </p>


                                                <h2 style="font-size: 20px; margin-top: 30px;color: black;">
                                                    Screening
                                                </h2>


                                                <p style="text-indent:2em;">
                                                    Transformants were selected from the mutational library plates, and each transformant was added with 1ml LBL medium (peptone 10g/L, yeast extract 5 g/L, sodium chloride 5g/L, 0.5 mg/ml ascorbic acid, pH=7.45, 100 μg/ mL amycin, 33 μg/ml chloramphenicol)
                                                    per well. 96-deep well plate was centrifuged at 300 rpm for 12h to prepare saturated bacterial solution. The saturated bacterial solution was then diluted 1:50 into a new 96-deep well plate containing
                                                    1ml antibiotic-free LBL medium per well and incubated at 300rmp for 2h in shock. At this time, the OD value of bacterial solution was close to 0.4, 100 μL of DOPA solution (10 mM/L L-DOPA, 0.5mg/ml ascorbic
                                                    acid) was added to each well, and then the culture was incubated for 2 hours with 300 rmp shock. Then 100ul DAM (5 g/L peptone, 2.5 g/L yeast extract, 5 g/L sodium chloride, 0.5 mg/ml ascorbic acid,
                                                    10ml L-DOPA, 25uM/L kanamycin, 0.23 μM/L, 2.96 g/L IPTG) was added to each well. The incubation was continued at 300rmp for 4h to obtain the induced bacterial solution.
                                                </p>
                                                <p style="text-indent:2em;">The induced bacterial solution was centrifuged at 3000 rmp for 30 min, and the supernatant was discarded. Each well was blown and resuspended with 200ul cold PBS buffer, and 100 μL was taken into a 96-well
                                                    fluorescence plate and measured with a microplate reader. Measurement indicators include Ex=450nm/Em=500nm fluorescence, Ex=535nm/Em=585nm fluorescence and OD 600 light absorption value.
                                                </p>
                                                <h2 style="font-size: 20px; margin-top: 30px;color: black;">
                                                    Characterization
                                                </h2>
                                                <h3 style="font-size: 17px; margin-top: 30px;color: black;">Analog analysis
                                                </h3>

                                                <p style="text-indent:2em;">
                                                    For the mutant, we modeled it by AlphaFold2. The simulation process was completed on the cloud server (Shenzhen Bkunyun Cloudcomputing Co.,Ltd.). After the simulation, we selected the model with the highest score as the protein model for our next molecular
                                                    dynamics simulation. The molecular dynamics simulations were done using Gromacs, Among them, Energy minimization, Ensemble Equilibration and final molecular dynamics simulation was completed on the cloud
                                                    server (Shenzhen Bkunyun Cloudcomputing Co.,Ltd.), and the remaining steps were performed locally. In the simulation, OPLS-AA/L All-Atom Force Field (2001 Aminoacid Dihedrals) was selected as the force
                                                    field, and SPC/E Water Force field was selected as the water molecular force field.
                                                </p>
                                                <h3 style="font-size: 17px; margin-top: 30px;color: black;">Validation test
                                                </h3>

                                                <p style="text-indent:2em;">
                                                    The mutants with high orange fluorescence value were selected and the plasmids were transferred into the competent state again, and the fluorescence value was measured whether it was consistent with that before.
                                                </p>

                                    </div>



                                    <div class="p4 h3" id="h3">
                                        </br>
                                        <h1 style="text-align:left;line-height: 30px; ">
                                            Methods of PACE </h1>
                                        <h2 style="font-size: 20px; margin-top: 30px;color: black;">
                                            Golden Gate cloning
                                        </h2>


                                        <p style="text-indent:2em;">
                                            SP was cloned by Golden Gate assembly.</p>
                                        <p>1. For Golden Gate assembly，SapI (New England BioLabs), was used as the type IIS restriction enzymes along with T4 DNA ligase (New England BioLabs). </p>
                                        <p>2. Typical assemblies contained final concentrations of ~0.5–2 ng kb−1 µl−1 plasmids, with a ~2/1 ratio of donor to acceptor plasmids (splitC and splitD).
                                        </p>
                                        <p>3. Assemblies were thermally cycled (5 min 37℃ , 5 min 16℃ for 30 cycles, 5 min 60°C ,4℃) and followed by transformation into chemically competent cells (Steps 4-7).
                                        </p>
                                        <h2 style="font-size: 20px; margin-top: 30px;color: black;">
                                            Phage cloning
                                        </h2>


                                        <p style="text-indent:2em;">
                                            4. Transform phage genomes assembled through Golden Gate cloning method into S2208 cells and substituting the plasmid used in Step 1 with the entire yield of assembled phage genome.
                                        </p>
                                        <p>5. After the heat shock step, put the cell mixture back on ice and stand for 2-3 min.
                                        </p>
                                        <p>6. Dilute the cell mixture into 10 ml of antibiotic-free 2× YT liquid media in a culture tube.
                                        </p>
                                        <p>7. Pellet 2 ml of the resultant culture by centrifugation at 8000 g for 3 min.
                                        </p>
                                        <p>8. Collect and filter the supernatant using a 3-ml syringe fitted with a 13-mm, 0.22-μm PVDF syringe filter to remove residual cells. Collected phage may be clonally isolated (Steps 23–25) using plaque assays (Steps
                                            18–22) and sequenced (Steps 26–29).
                                        </p>
                                        <h2 style="font-size: 20px; margin-top: 30px;color: black;">
                                            Activity-independent phage plaque assays
                                        </h2>


                                        <p style="text-indent:2em;">
                                            9. Dilute a saturated culture of S2208 cells in 2×YT liquid media supplemented with ampicillin and grow to an OD600 of 0.6–0.9 at 37℃ in a biological shaker.</p>
                                        <p>10. Dilute phage serially in water in three 100-fold increments to yield four total samples (undiluted and 102-, 104- and 106-fold diluted). </p>

                                        <p>11. Transfer 10 μl of each dilution into microcentrifuge tubes. </p>

                                        <p>1.2× YT top agar. To each phage dilution, add 150 μl of diluted S2208, add Bluo-Gal to 2× YT top agar at a final concentration of 0.04%, stand for 2-3 min, add 1 ml of warm (~55 ℃)2×YT top agar. </p>

                                        <p>12. After mixing, plate each phage dilution onto one quadrant of a Petri X-plate containing 2 ml of solidified 2×YT agar in each quadrant. </p>

                                        <p>13. Incubate plates overnight at 37 °C. </p>

                                        <p>14. Calculate the phage titer by the following formula: titer (in pfu/ml) = (# of plaques in quadrant) × (dilution factor of quadrant) × 100.
                                        </p>



                                        <h2 style="font-size: 20px; margin-top: 30px;color: black;">
                                            Isolation of clonal phage
                                        </h2>


                                        <p style="text-indent:2em;">

                                            15. Pick a single plaque from a plaque assay plate by touching a P10 pipette tip to the surface of the top agar.
                                        </p>
                                        <p>16. Place the pipette tip into 2-3 ml of DRM and grow at 37 °C in a biological shaker for 16-20 h.
                                        </p>
                                        <p>17. Pellet the cells by centrifuging at 8000 g for 2 min, collect and filter the supernatant using a 3-ml syringe fitted with a 13-mm, 0.22-μm PVDF syringe filter to remove residual cells.
                                        </p>
                                        <p>18. Determine the phage titer using an activity-independent phage plaque assay (Steps 9-15)


                                        </p>

                                        <h2 style="font-size: 20px; margin-top: 30px;color: black;">
                                            Sequencing clonal phage
                                        </h2>

                                        <p style="text-indent:2em;">

                                            19. Pick a single plaque from a plaque assay plate by touching a P10 pipette tip to the surface of the top agar.
                                        </p>
                                        <p>20. Amplify the phage DNA by using PCR amplify the POI insert on the phage (Place the tip into a PCR reaction mixture. Then, let the pipette tip from Step 20 sit in the reaction for at least 30 s before removing.
                                            After running the PCR reaction, submit PCR product directly for Sanger sequencing.).
                                        </p>
                                        <p>21. Place the pipette tip from (Step 16) in a culture tube containing 3 ml of DRM and incubate in a biological shaker at 37℃ for 16-20 h.
                                        </p>
                                        <p>22. Pellet the cells by centrifuging at 8,000g for 2 min and then collect and filter the supernatant using a 3-ml syringe fitted with a 13-mm, 0.22-μm PVDF syringe filter to remove residual cells.

                                        </p>


                                        <h2 style="font-size: 20px; margin-top: 30px;color: black;">
                                            MP6 mutation rate assay
                                        </h2>

                                        <p style="text-indent:2em;">
                                            23. MP6 was transformed into S2060 and plated on Chl-resistant 2×YT plates containing 100mM D-Glucose.
                                        </p>
                                        <p>24. Monoclones were picked and shaken till saturation in 2×YT medium containing 25 mM D-Glu and Chl;
                                        </p>
                                        <p>25. The above bacterial solution was diluted 103-fold and grown to OD600 of 0.5-0.7. Grown to mid-log phase.
                                        </p>
                                        <p>26. Samples were divided into two: one with 25 mM D-Glucose and the other with 25 mM arabinose, and incubated for 12h until saturation was induced.
                                        </p>
                                        <p>27. The saturated solution was diluted by gradient and applied to 2×YT plates containing 100 mM D-Glucose and Chl and the saturated solution was applied to 2×YT plates containing 100 mM D-Glucose and Rif 37°C for
                                            18-24 h.
                                        </p>
                                        <p>28. The number of colonies of each culture on D-Glucose ± Rif plates was counted. Formula for calculating MP6 mutation rate (tbp: substituents per bp per generation): ubp = f/[R × ln(N//N0)].


                                        </p>
                                        <h2 style="font-size:20px; margin-top: 30px;color: black;">
                                            Luciferase assay
                                        </h2>

                                        <p style="text-indent:2em;">

                                            29. AP was transformed into S2060 competent cells（Steps 4-7）. It coupled bacterial luciferase (LuxAB) with <i>gⅢ</i> translationally.
                                        </p>
                                        <p>30. Pick a single colony into DRM supplemented with Ampicillin and Streptomycin and incubate it in a biological shaker at 37℃ for 16-20 h.
                                        </p>
                                        <p>31. Make 100 times dilution of the saturated culture of host cells into DRM and incubate in a biological shaker at 37℃ until the OD
                                            <SUB>600</sub> of diluted culture reaches 0.4-0.6.
                                        </p>
                                        <p>32. Infect the host cells with phage. Incubate cells in a biological shaker at 37℃ for 1-2 h.
                                        </p>
                                        <p>33. Read luminescence using a plate reader with the following settings:
                                        </p>
                                        <p>(1) Set temperature to 37℃.
                                        </p>
                                        <p>(2) Shake (orbital) for 10 s with 1-mm amplitude and 582 r.p.m.
                                        </p>
                                        <p>(3) Read absorbance at 600 nm (number flashes = 10, settle time = 0 ms).
                                        </p>
                                        <p> (4) Read luminescence (attenuation = none, settle time = 0 ms, integration time = 500 ms).


                                        </p>
                                    </div>




                                </aside>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        <div class="footer " style="position:absolute;right:0;left:0;margin:0 auto;width: 100%;height: 400px;background-color: black;margin-top: -200px;position: relative;">
            <div class="container" style="width:100%;height:100%;background-color: black;width: 100%;font-size: 14px;">
                <div class="footer-img" style="position:absolute;left: 0;">
                    <img src="./image/footer.jpg" width="600px" height="230px">
                </div>


                <div class="footer-content" style="text-align: center;right: 0;height: 100%;width: 50%;background-color: black;top: 0;">

                    <ul class="footer-content-ul">
                        <li class="first">
                            <a href="./index.html" style="font-size: 21px;">HOME</a>
                        </li>


                    </ul>

                    <ul>
                        <li class="first">
                            <a href="./description.html" style="font-size: 21px;">PROJECT</a>

                        </li>
                        <li>
                            <a href="./description.html">Description</a>
                        </li>
                        <li>
                            <a href="./design.html">Design</a>

                        </li>
                        <li>
                            <a href="./results.html">Results</a>
                        </li>
                        <li>
                            <a href="./supplement.html">Supplement</a>
                        </li>

                        <li>
                            <a href="./protocol.html">Protocol</a>
                        </li>


                    </ul>

                    <ul class="footer-content-ul">
                        <li class="first">
                            <a href="./lab_notebook.html" style="font-size: 18px;">LAB NOTEBOOK</a>
                        </li>
                    </ul>

                    <ul class="footer-content-ul">
                        <li class="first">
                            <a href="./team.html" style="font-size: 21px;">TEAM</a>
                        </li>
                        <li>
                            <a href="./team.html">Team</a>
                        </li>
                        <li>
                            <a href="./collaboration.html">Collaboration</a>
                        </li>
                    </ul>

                </div>

                <div class="footer-row">
                    <div class="col-md-12 " style="margin-top: -30px; ">
                        <p style="text-align: center;font-size: 16px; color: white;">Copyright © 2022 NAU All rights reserved.
                        </p>
                    </div>
                </div>


            </div>
        </div>
        <script src="./js/longli.js"></script>

</html>