<!DOCTYPE html>
<html lang="en" dir="ltr" class="client-nojs">

<head>
    <meta charset="UTF-8" />
    <title>Results</title>
    <link rel="stylesheet" type="text/css" href="css/nav.css">
    <link rel="stylesheet" type="text/css" href="./css/new1.css">
    <link rel="stylesheet" type="text/css" href="css/painter.css">
    <link rel="stylesheet" type="text/css" href="./css/page/results_painter.css">
    <link rel="stylesheet" type="text/css" href="./css/page/results.css">
    <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.5.1.min.js"></script>
    <script>
        window.addEventListener('scroll', function() {
            let top = window.scrollY
            console.log(top)
            if (top > 200) {
                $('.main').css('transform', 'translateY(-300px)')
            } else {
                $('.main').css('transform', 'translateY(0%)')
            }
        })
    </script>
</head>

<body>
    <div id="loading">
        <div class='loadingX' style="background-color: #abc2ea;">
            <img src="./image/lodding_3.gif" height=100% width=100%>
        </div>
    </div>

    <script>
        document.onreadystatechange = function() {

            if (document.readyState === "complete") {

                setTimeout(
                    function() {
                        // document.querySelector("#loading").style.opacity = "0%";
                        document.querySelector("#loading").style.display = "none";
                        document.querySelector("#globalWrapper").style.display = "block";
                    }, 3000)
            }
        }
    </script>

    <div id='globalWrapper'>
        <div id="content" role="main" style="background-color:  white">
            <div id="HQ_page">
                <section class="menu-section-area">
                    <!-- Navigation -->
                    <nav class="menu">
                        <div style="height: 0px;width: 0px;position: relative;">
                            <img src="./image/logo.png" width="50px" height="50px" style="position:absolute;top:-14px;">
                        </div>

                        <ol>
                            <li class="menu-item"><a href="./index.html">Home</a></li>

                            <li class="menu-item">
                                <a href="#0">Project
                                    <svg version="1.1" class="plus-icon" xmlns="http://www.w3.org/2000/svg"
                                        xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 18 18">
                                        <line fill="none" stroke-width="4" stroke-linecap="round" stroke-miterlimit="10"
                                            x1="10" y1="9" x2="17" y2="9" />
                                        <line fill="none" stroke-width="4" stroke-linecap="round" stroke-miterlimit="10"
                                            x1="9" y1="9" x2="9" y2="1" />
                                        <g id="lineGroup_1">
                                            <line fill="none" stroke-width="4" stroke-linecap="round"
                                                stroke-miterlimit="10" x1="1" y1="9" x2="8" y2="9" />
                                            <line fill="none" stroke-width="4" stroke-linecap="round"
                                                stroke-miterlimit="10" x1="9" y1="17" x2="9" y2="9" />
                                        </g>
                                    </svg>
                                </a>
                                <ol class="sub-menu">
                                    <li class="sub-menu-item"><a href="./description.html">Description</a></li>
                                    <li class="sub-menu-item"><a href="./design.html">Design</a></li>
                                    <li class="sub-menu-item"><a href="./results.html">Results</a></li>
                                    <li class="sub-menu-item"><a href="./supplement.html">Supplement</a></li>

                                    <li class="sub-menu-item"><a href="./protocol.html">Protocol</a></li>
                                </ol>
                            </li>
                            <li class="menu-item"><a href="./lab_notebook.html">Lab notebook</a></li>
                            <li class="menu-item">
                                <a href="#0">Team
                                    <svg version="1.1" class="plus-icon" xmlns="http://www.w3.org/2000/svg"
                                        xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 18 18">
                                        <line fill="none" stroke-width="4" stroke-linecap="round" stroke-miterlimit="10"
                                            x1="10" y1="9" x2="17" y2="9" />
                                        <line fill="none" stroke-width="4" stroke-linecap="round" stroke-miterlimit="10"
                                            x1="9" y1="9" x2="9" y2="1" />
                                        <g id="lineGroup_1">
                                            <line fill="none" stroke-width="4" stroke-linecap="round"
                                                stroke-miterlimit="10" x1="1" y1="9" x2="8" y2="9" />
                                            <line fill="none" stroke-width="4" stroke-linecap="round"
                                                stroke-miterlimit="10" x1="9" y1="17" x2="9" y2="9" />
                                        </g>
                                    </svg>
                                </a>
                                <ol class="sub-menu">
                                    <li class="sub-menu-item"><a href="./team.html">Team</a></li>
                                    <li class="sub-menu-item"><a href="./collaboration.html">Collaboration</a></li>

                                </ol>
                            </li>



                    </nav>
                    <!-- Navigation -->

                </section>
                <div class="img1"></div>

                <div class="lnav">
                    <ul class="navul" style="margin-left: 0;padding: 0;">
                        <li id="nav1" class="uli cntli" style="height: 60px;line-height: 1.5em;">
                            <a href="#p1">Bioinformatics Analysis</a>
                        </li>
                        <li id="nav2" class="uli cntli" style="height: 60px;line-height: 1.5em;">
                            <a href="#p2">Screening by Fluorescence</a>
                        </li>
                        <li id="nav3" class="uli cntli" style="height: 50px;">
                            <a href="#p3">PACE</a>
                        </li>
                        <!-- <li id="nav4" class="uli cntli">
                            <a href="#p5">Note</a>
                        </li> -->
                        <li id="nav4" class="uli cntli" style="height: 50px;">
                            <a href="#p4">Reference</a>
                        </li>


                    </ul>





                </div>


                <div class="content">
                    <div class="main" style="color: black;word-wrap: break-word;word-break: normal;">
                        <aside>
                            <div class="p1" id="p1">
                                </br>
                                <h1 style="text-align:left;font-size: 25px;line-height: 30px; ">Bioinformatics Analysis
                                </h1>

                                <p style="text-indent:2em;">Four mutational residues (Y32L, A67S, H70N, A167Q) were introduced to Gen 2 and the mutant was named "Mutpre" (mutant prediction). The structures of Gen 2 and Mutpre predicted by AlphaFold2 were aligned and the consequent
                                    RMSD (Root Mean Square Deviation) was 2. 01, which means they shared similar structures (RMSD
                                    < 3). Analysis of the four mutational residues indicates that they are closed to each other in tertiary structure, though not continuous in primary structure. The four residues formed a pocket together, among which the Y32 itself is a binding site of
                                        tyrosine. The molecular dynamic simulation of the two proteins shows that, Mutpre seems to be less stable than Gen 2 (the former has larger RMSD). Mutpre and Gen 2 shared a similar level of compactness, for they shared
                                        similar fluctuating range of Rg, radius of gyration. Most importantly, four mutational residues increased the flexibility of Mutpre compared to Gen 2 globally, indicated by RMSF (Root Mean Square Fluctuation). The flexibility
                                        of some local regions are also significantly increased. The increase of global and local flexibility may indicate higher catalytic activity. </p>

                                        </br>

                                        <div style="display: flex;justify-content: center;align-items: center;">
                                            <img src="./image/results-1.jpg" width="600px" height="400px" style="">
                                        </div>

                                        </br>


                                        <p><b>Figure 1. Results of modeling and MD simulation. (a) </b>Alignment of Gen 2 and Mutpre. The grey structure shows Gen 2 and the pink structure shows Mutpre. Y32, A67, H70, A167 in Gen 2 were emphasized by red
                                            sticks and balls, while 32Y, 67A, 70H, 167A in Mutpre were by grey. (b) RMSD of Gen 2 and Mutpre during 1 ns simulation. (c) Radius of gyration of Gen2 and Mutpre during 1 ns simulation. (d) RMSF of Gen 2 and
                                            Mutpre per residue. </p>



                            </div>
                            </br>
                            </br>

                            <div class="p2" style="" id="p2">
                                <h1 style="text-align:left;font-size: 25px;line-height: 30px; ">Screening by Fluorescence
                                </h1>

                                <p style="text-indent:2em;">Mutation libraries were constructed by error-prone PCR and site-directed mutagenesis. To determine the optimal conditions of error-prone PCR, a series of annealing temperature gradients and StarMut Enhancer gradients were
                                    set, and the optimal error-prone reaction system was determined based on the results of agarose gel electrophoresis. The final error-prone PCR conditions used were 53. 1℃ (annealing temperature) and 6 μL enhancer. </p>

                                </br>

                                <div style="display: flex;justify-content: center;align-items: center;">
                                    <img src="./image/results-2.png" width="500px" height="400px" style="">
                                </div>

                                </br>


                                <p><b>Figure 2. Pre-experiments to explore best error-prone PCR condition. </b>Set the StarMut Enhancer volume gradients for error-prone PCR, the enhancer volumes were 2μL, 4μL and 6μL in order. 6μL enhancer in 20 µL system
                                    with appropriate mutation rate was finally chosen.
                                </p>

                                <p style="text-indent:2em;">
                                    The mutated fragment was homologously recombined with the linearized vector p15A-pBAD, and transferred into C321 ΔA muts-T7. The mutation rate is 3-18 mutant bases per kb. Currently, the library constructed by error-prone PCR has contained approximately
                                    1. 5 × 10<sup>4</sup> mutants.
                                </p>

                                <p style="text-indent:2em;">
                                    Four amino acid mutation sites were predicted based on Hotspot Wizard software: Y32, A67, A70 and A167. We designed three pairs of IUB code-containing degenerate primers and used Gen 2 as a template to construct the library. The library constructed by
                                    site-directed mutation has contained approximately 500 mutants.
                                </p>

                                <p style="text-indent:2em;">
                                    A screening platform that measures the degree of incorporation of L-DOPA was established, and we quantified its efficiency and specificity. TyrRS specifically recognizes tyrosine and catalyzes the formation of tyrosyl-tRNA. To avoid affecting the normal
                                    function of the original TyrRS in cells, we applied the modified TyrRS to the orthogonal translation system. This modified TyrRS catalyzes the aminoacylation of an artificial tRNA whose anticodon is UAG. Therefore,
                                    <i>E. coli</i> C321 ΔA muts-T7 strain was chosen as the host, in whose genome the UAG codons have been replaced by other stop codon, in order to avoid the incorporation of Tyr in UAG anticodon. The orthogonal TyrRS
                                    and modified strain enable the incorporation of noncanonical amino acids without interfering with the normal growth of <i>E. coli</i>.
                                </p>

                                <p style="text-indent:2em;">
                                    In order to select aaRS variants with potential capability of L-DOPA incorporation, sfGFP Y66-S205C containing the L-DOPA incorporation was applied as a reporter. It can emit green fluorescence (absorption peak at Ex=450 nm/Em=500 nm) with residue Y66
                                    while emitting orange fluorescence (absorption peak at Ex=535 nm/Em=585 nm) with residue DOPA66 . The intensity of orange fluorescence is positively proportional to the increase of L-DOPA incorporation. Thereby aaRS
                                    variants’ specificity and activity of binding L-DOPA could be quantified.
                                </p>

                                </br>

                                <div style="display: flex;justify-content: center;align-items: center;">
                                    <img src="./image/results-3.jpg" width="650px" height="350px" style="">
                                </div>

                                </br>

                                <p style="">
                                    <b>Figure 3. (a) </b>After incubation with inducer for 8 hours, we resuspended the culture of transformants and plotted the emitted fluorescence intensity (Ex = 440-560 nm). (b) Emission spectrum of culture of transformants
                                    (Ex = 450 nm). The emission peak is around 500 nm. (c) Emission spectrum of culture of transformants (Ex = 535 nm). The emission peak is around 585 nm.
                                </p>

                                <p style="text-indent:2em;">
                                    To ensure that sfGFP could be expressed with the existence of aaRS, Gen 2 operated by the arabinose promoter was integrated into the pET28a vector; sfgfp operated by the IPTG operon was integrated into the pBAD33 vector. The two plasmids mentioned above
                                    were transformed into C321 ΔA muts-T7 to build a complete pathway.
                                </p>

                                </br>

                                <div style="display: flex;justify-content: center;align-items: center;">
                                    <img src="./image/results-5.png" width="350px" height="260px" style="">
                                </div>

                                </br>

                                <p style="">
                                    <b>Figure. 4 </b>The map of sfGFP Y66-S205C. This plasmid contains the sfGFP-Y66DOPA-S205C sequence and the IPTG operon used to manipulate it.
                                </p>

                                </br>

                                <div style="display: flex;justify-content: center;align-items: center;">
                                    <img src="./image/results-4.jpg" width="320px" height="250px" style="">
                                </div>

                                </br>

                                <p style="">
                                    <b>Figure. 5 </b>The map of p15A-PBAD-Gen2-tRNACUA opt. This plasmid contains the MjTYR-Gen2 sequence and the arabinose promoter used to manipulate it, as well as the modified tRNA(with UAG anticode) sequence used to
                                    express substrate for aminoacylation reactions.
                                </p>

                                <p style="text-indent:2em;">
                                    Pre-experiments were conducted to determine a stable induction culture protocol by setting different culture conditions, such as L-DOPA concentration, media type, presence or absence of chloramphenicol, etc. When DOPA concentration was 0. 78 mM/L and
                                    chloramphenicol was present, the leakiness on the lac operon and arabinose promoter was relatively low. To obtain intensive fluorescence, the DOPA is induced to incorporate into sfGFP with the culturing of LBL and DAM
                                    medium. Finally, we applied 0. 78 mM/L L-DOPA, LBL and DAM medium to obtain low leakage expression and the intensive fluorescence level.
                                </p>

                                </br>

                                <div style="display: flex;justify-content: center;align-items: center;">
                                    <img src="./image/results-6.png" width="620px" height="550px" style="">
                                </div>

                                </br>

                                <p style="text-indent:0em;">
                                    <b>Figure 6. (a) </b>Three control groups were set: no induction, only arabinose, only IPTG. The experimental group was added with both Ara and IPTG. The base concentration of L-DOPA was 0. 52 mM/L, on which we diluted
                                    to 0. 8-10 times. We found that 1-4 times of L-DOPA worked better, and 2 times of L-DOPA acted best. (b) The control groups were added to all inducers separately without DAM, and the experimental group was added to
                                    DAM. The Mean Fluorescence Intensity (MFI)/OD<sub>600</sub> of the experimental group was about 2 times about that of the control groups.
                                </p>
                                </br>

                                <div style="display: flex;justify-content: center;align-items: center;">
                                    <img src="./image/results-7.png" width="670px" height="400px" style="">
                                </div>

                                </br>

                                <p style="">
                                    <b>Figure 7. </b>Overview of mutation, selection and characterization strategy in this study. Gen 2 was mutated by error-prone PCR and site-directed mutagenesis. We use homologous recombination to construct mutation
                                    library. Mutants were then selected by fluorescence with plate reader.
                                </p>

                                <p style="text-indent:2em;">
                                    Our mutation library contains about 1. 5×10<sup>4</sup> positive transformants in total. The single clones were cultured, induced, and measured 30 variants were screened by fluorescence assay. We measure the fluorescence
                                    after induced expression of fluorescent protein to screen out mutant strains with high orange MFI. In general, the fluorescence intensity is positively correlated with the reproduction of the strains (the expression
                                    amount of sfGFP) and the L-DOPA incorporation efficiency. However, considering the influence of background fluorescence intensity on instrument detection, we usually consider data with low OD<sub>600</sub> to be unreliable.
                                    Therefore, the variants with high MFI/OD<sub>600</sub> but too low OD<sub>600</sub> will be firstly eliminated.
                                </p>

                                </br>

                                <div style="display: flex;justify-content: center;align-items: center;">
                                    <img src="./image/results-8.png" width="500px" height="300px" style="">
                                </div>

                                </br>

                                <p style="">
                                    <b>Figure. 8 </b>Heatmap of partial mutation library. Variants mutated by error-prone PCR or site-directed mutagenesis (SDM) were labled on the raw title. Mutatants with high MFI/OD<sub>600</sub> but too low OD<sub>600</sub>                                    were gray and placed "X" through on.
                                </p>

                                <p style="text-indent:2em;">
                                    The variants MFI/OD<sub>600</sub> was higher than that of the control group, and the colony reproduction was at a reasonable level (1. 0
                                    < OD<sub>600</sub>
                                        < 1. 4). Statistically, it was found that the MFI / OD<sub>600</sub> of these variants was concentrated in the range of 1. 2-2. 0 times that of the control group. They are considered benign mutants to bind L-DOPA strongly. These variants were sequenced to further explore
                                            the exact mutation points. So far, we have not obtained the final sequencing data. In the future, the necessary analysis to this section will be added. </p>




                            </div>
                            <div class="p3" id="p3">
                                </br>
                                <h1 style="text-align:left;font-size: 25px;line-height: 30px; ">Phage-assisted
                                    <br />Continuous Evolution (PACE)</h1>

                                <p style="text-indent:2em;">Currently, we have successfully constructed the plasmids (SP, AP and CP) required for PACE.
                                    </br>
                                </p>
                                <div style="display: flex;justify-content: center;align-items: center;">
                                    <img src="./image/results/r-1.jpg" width="300px" height="230px" style="">
                                </div>
                                </br>
                                <p><b>Figure 9. The map of selection phage (SP). </b>The SP plasmid is based on the M13 wild-type phage with the
                                    <I>gⅢ</I> removed, plus the tyrosine aminoacyl tRNA synthetase to be mutated and an artificial RBS sequence.
                                </p>
                                <div style="display: flex;justify-content: center;align-items: center;">
                                    <img src="./image/results/r-2.jpg" width="350px" height="300px" style="">
                                </div>

                                </br>

                                <p><b>
                                        Figure 10. The map of accessory plasmid (AP).
                                    </b>The AP plasmid includes a tRNA sequence that recognises the TAG codon and the
                                    <i>gⅢ</i> and LuxAB genes controlled by the T7 promoter.

                                </p>

                                <div style="display: flex;justify-content: center;align-items: center;">
                                    <img src="./image/results/r-3.png" width="320px" height="260px" style="">
                                </div>

                                </br>

                                <p><b>

                                        Figure 11. The map of complementary plasmid (CP).

                                    </b>The CP plasmid contains the T7 RNAP, which contains two amber termination mutations.
                                </p>
                                <p>
                                    All plasmids constructed are verified by sequencing. In addition, we validated the SP by virus plaque formation and validated the AP by luciferase characterization.

                                </p>

                                <div style="display: flex;justify-content: center;align-items: center;">
                                    <img src="./image/results/ab.jpg" width="800px" height="400px" style="">
                                </div>

                                </br>


                                <p><b>


                                        Figure 12. Previous results of PACE. (a)


                                    </b>Wild-type M13 phage; from the upper right quadrant, upper left quadrant, lower right quadrant, and lower left quadrant, the phage concentrations are 100, 10-2, 10-4, 10-6 in order, and the number of phages in appropriate
                                    in quadrants can be used to quantify the titer. (b) Luciferase characterization. TyrRS ligates L-DOPA to the amber repressor tRNA, enabling complete translation of T7 RNAP and access to the protein structure. T7 RNAP
                                    production activates the T7 promoter, leading to the expression of the downstream luciferase. That is, it can be used to monitor the intensity of <i>gⅢ</i> expression, whether the lines are coupled, and the amount of
                                    amber repressor introduced in T7 RNAP.

                                </p>

                                <p style="text-indent:2em;">The PACE device is partially based on the work of Liu and co-workers
                                    <sup>1</sup>. Due to the differences in industrial product standards between the US and China, we have built the complete device using partially functionally identical Chinese made equipment.
                                </p>
                                <p style="text-indent:2em;">
                                    The tubing used in the device and its matching Luer fittings and other accessories were all from Baoding Longer Precision Pump Company Co., Ltd. The two peristaltic pumps used were the BT100M model from Baoding Chuang Rui Precision Pump Co., Ltd. with
                                    the DG-N model pump head. And the two syringe pumps were the LD-P2020II model from Shanghai Lande Medical Devices Co.,Ltd. The rest of the small parts were sourced from Taobao.

                                </p>
                                <p style="text-indent:2em;">Once the equipment and materials were available, we assembled and commissioned the device and confirmed that the whole system was operational. Two peristaltic pumps controlled the inflow and outflow of media from the chemostat,
                                    and the inflow and outflow of waste from the chemostat cell culture in the lagoons. And two syringe pumps controlled the injection of arabinose solution into the two lagoons. The apparatus are shown below.
                                </p>
                                <div style="display: flex;justify-content: center;align-items: center;">
                                    <img src="./image/results/r-5.jpg" width="500px" height="300px" style="">
                                </div>

                                </br>

                                <p><b>Figure 13. Photo of our PACE apparatus.</b> The devices from left to right are Peristaltic Pump 1, Chemostat, Peristaltic Pump 2, Lagoons 1 and 2, and Syringe Pumps 1 and 2. The medium flows continuously throughout the
                                    system through the bucket at the rear. During the experiment, the parts other than the electronics will be placed in an incubator at around 37°C.

                                </p>
                                <p style="text-indent:2em;">
                                    In the meantime, we used the <i>E.coli</i> S2208 to validate the chemostat’s function and enable it to output a constant fluid with the absorbance (OD<sub>600</sub>) of ~0.6 for 16 hours.
                                </p>

                                <div style="display: flex;justify-content: center;align-items: center;">
                                    <img src="./image/results/r-6.png" width="900px" height="400px" style="">
                                </div>

                                </br>

                                <p><b>
                                        Figure 14. Dynamic changes in the absorbance of the culture liquid during the
                                        commissioning process.
                                    </b> In order to make it easier to monitor the operating status of the chemostat after the device is in stable operation, we have designed an Arduino-based device for the real-time determination of the 600 nm absorbance
                                    of the bacterial solution in the chemostat, taking into account the work of Lin<sup>2</sup>. The device measures the absorption of 940 nm light-emitting diode light by the bacterial solution and converts it to an approximate
                                    value of OD<sub>600</sub> against a calibration curve determined in advance, thus providing information for the working state of the chemostat.
                                </p>




                                <div style="display: flex;justify-content: center;align-items: center;">
                                    <video src="./image/results/lod.mp4" controls="controls" loop="loop" style="width: 400px;" muted autoplay></video>
                                </div>

                                </br>

                                <p><b>
                                        Figure 15. The device we designed for measuring absorbance. </b> We had encountered great difficulties in the SP plasmid construction part due to lack of experience. Now we are happy to see that the system is taking
                                    shape. However, we still have some work to do before the PACE system is officially running. We will continue to experiment and make efforts for more progress in the future.
                                </p>

                                <p style="text-indent:2em;">Source Code of the device for measuring absorbance is available in <a href="https://github.com/Station724/real-time_determination_of_the-_absorbance">GitHub</a>.


                                    </br>
                                </p>
                            </div>


                            <div class="p4" id="p4">
                                </br>
                                <h1 style="text-align:left;font-size: 25px;line-height: 30px; ">Reference</h1>

                                <p>1 S. M. Miller, T. Wang, D. R. Liu, Phage-assisted continuous and non-continuous evolution. Nature Protocols 15, 4101-4127 (2020). <br /> https://10.1038/s41596-020-00410-3
                                </p>
                                <P> 2 D.-S. Lin, C.-H. Lee, Y.-T. Yang, Wireless bioreactor for anaerobic cultivation of bacteria. Biotechnology Progress 36, (2020). <br /> https://10.1002/btpr.3009


                                </p>


                            </div>


                            </br>
                            </br>


                        </aside>
                    </div>
                </div>
            </div>
        </div>
        <div class="footer " style="position:absolute;right:0;left:0;margin:0 auto;width: 100%;height: 400px;background-color: black;margin-top: -200px;position: relative;">
            <div class="container" style="width:100%;height:100%;background-color: black;width: 100%;font-size: 14px;">
                <div class="footer-img" style="position:absolute;left: 0;">
                    <img src="./image/footer.jpg" width="600px" height="230px">
                </div>


                <div class="footer-content" style="text-align: center;right: 0;height: 100%;width: 50%;background-color: black;top: 0;">

                    <ul class="footer-content-ul">
                        <li class="first">
                            <a href="./index.html" style="font-size: 21px;">HOME</a>
                        </li>


                    </ul>


                    <ul>
                        <li class="first">
                            <a href="./description.html" style="font-size: 21px;">PROJECT</a>

                        </li>
                        <li>
                            <a href="./description.html">Description</a>
                        </li>
                        <li>
                            <a href="./design.html">Design</a>

                        </li>
                        <li>
                            <a href="./results.html">Results</a>
                        </li>
                        <li>
                            <a href="./supplement.html">Supplement</a>
                        </li>

                        <li>
                            <a href="./protocol.html">Protocol</a>
                        </li>


                    </ul>

                    <ul class="footer-content-ul">
                        <li class="first">
                            <a href="./lab_notebook.html" style="font-size: 18px;">LAB NOTEBOOK</a>
                        </li>
                    </ul>

                    <ul class="footer-content-ul">
                        <li class="first">
                            <a href="./team.html" style="font-size: 21px;">TEAM</a>
                        </li>
                        <li>
                            <a href="./team.html">Team</a>
                        </li>
                        <li>
                            <a href="./collaboration.html">Collaboration</a>
                        </li>
                    </ul>

                </div>

                <div class="footer-row">
                    <div class="col-md-12 " style="margin-top: -30px; ">
                        <p style="text-align: center;font-size: 16px; color: white;">Copyright © 2022 NAU All rights reserved.
                        </p>
                    </div>
                </div>


            </div>
        </div>
        <script src="./js/longli.js"></script>

</html>